docu/Concepts/TechnicalRequirements/UC_X-Com-Tx_per_Link.md

```diff
@@ -0,0 +1,116 @@
+* If the expiration time of the code is not exceeded
+
+Only if all validation checks are successful the community will start the _disbursement process_ with the 1st step to select the recipient's community.
+
+#### Community-Selection on sender-side
+
+In consequence the first shown page for the recipient-user must offer a community-selection to yield the decision for the recipient community.
+
+The first page the user will see on activating the redeem-link looks like quite the following page:
+
+![](./image/redeemlink-page_with-community-selection.png)
+
+To support a cross community redeem-link activation this page is extended with the possibility to select a _receiver-community_. An UI-component will present a list of all known, verified and authenticated communities the sender's home-community is connected over the federation. Per default the _sender-community_ is selected, the _register-_ and _login_-links are visible and the shown button to switch to the selected community is hidden.
+
+Only if the user select a foreign community out of the community-list the _switch-community_-button is visible and both _register-_ and _login_-links are hidden. This force the user to first switch to the _receiver-community_ before he could login or register himself.
+
+The backend offers for the initialization of the community-selection-list the graphql query:
+
+```
+CommunityResolver.authenticatedCommunities :Promise Community[] 
+```
+
+returning only Community-Entities with a defined `uuid` and a given `authenticatedAt` Date by the following type definition:
+
+```
+Community {
+  id: number
+  foreign: boolean
+  name: string
+  description: string
+  url: string
+  creationDate: Date
+  uuid: string
+  authenticatedAt: Date
+  gmsApiKey: string
+}
+```
+
+With the selection of a community-entry from this list the dialog evaluats the _foreign_-flag of the selected community and shows or hide the _community-switch-button_ and the _login-_ and _register_-links as described above.
+
+If the user doesn't change the default _sender_-community, the system goes on with the local login or registration page for a local `redeem-link activation`.
+
+In case of the recipient community will be a foreign community, the system has to prepare a request with a _Disbursement-JWT-Token_ as parameter to invoke on the foreign community the _disbursement process_ by following the pattern: `community-url of receiver-community`/redeem/`<Disbursement-JWT-Token`
+
+#### Disbursement JWT-Token
+
+This _Disbursement-JWT-Token_ contains all necessary information to start a _disbursement process_ on the foreign community. The payload of this token contains:
+
+* type of JWT-Token - here `redeem-activation`
+* the community-uuid of the sender-community
+* the gradidoID of the sender
+* the original code of the redeem-link
+* the alias if exists or the firstname of the sender
+* the amount of gradidos the sender will send
+* and the memo
```

Comment:
Why sending the data which should be in sender community db stored already alongside the redeem-code?
The only info needed together with the code is the receiver identification, isn't it?
Or should it support partly redeeming gradidos from link?